bin/pyactivate

```diff
@@ -119,6 +119,7 @@ def activate_venv(py_dir: Path, dev: bool = False) -> Path:
     ):
         os.environ["ARCHFLAGS"] = "-arch x86_64"
 
+    acquire_deps(venv_dir, "core")
```

Comment:
Can you add a comment to this effect?

```py
# The version of pip and setuptools installed into the virtualenv by default
# may have bugs like pypa/pip#9138 that cause it to e.g. fail to download
# wheels. So the first thing we do is upgrade these dependencies to a known
# version. This won't help if pip is so broken that it can't upgrade itself,
# but in that case there's nothing we can do.
```